xep-0434.xml

```diff
@@ -26,48 +29,57 @@
 	</dependencies>
 	<supersedes/>
 	<supersededby/>
-	<shortname>NOT_YET_ASSIGNED</shortname>
+	<shortname>None</shortname>
```

Comment:
Is there any specific reason to change this from `NOT_YET_ASSIGNED` to `None`?

Reply:
I am not familiar with the process of assigning a short name. Therefore, I thought I should enter `None` since the XEP has no short name. Could you tell me how it should be done?

Reply:
NOT_YET_ASSINGED is more or less the canonical String XEPs carry if there is not yet a shortname assigned. You can enter your preferred shortname here, usually the editor will check if the shortname is available and, if so, simply accept it.

Reply:
@melvo Yeah, please stick with `NOT_YET_ASSIGNED`. It is better to have a single "not assigned" value then a bunch of different ones. Easier to automate with that.

Reply:
Thanks for your explanations! I added a short name. If that is already in use, I would change it to `NOT_YET_ASSIGNED`.